pull_request_template.md

```diff
@@ -0,0 +1,20 @@
+<!-- 
+This PR template serves as a guide and reminder for contributors to ensure that the "blog" label is applied appropriately.
+Please make sure to apply the "blog" label to the pull request if it is related to a blog post so that it can be published to Slack. 
+-->
+
+## Description
+
+<!-- Add a description of the changes introduced by this PR -->
+
+## Related Issue
+
+<!-- If this PR is related to an issue, provide the issue number or a brief description -->
+
+## Checklist
+
+- [ ] I have applied the "blog" label to this pull request.
+- [ ] I have tested the changes locally.
+- [ ] I have reviewed the code changes.
+- [ ] I have updated the documentation, if necessary.
+- [ ] I have added appropriate tests, if applicable.
```

Comment:
I don't think a checklist is necessary at this time unless you feel strongly about it.

Reply:
Oh okay. I think it can be left out. Made a new push @tim-schilling